shell_scripts/idm_postprocessing.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#SBATCH --output=sbatch_logs/postprocess_results/postprocess_results_%A.log
+#SBATCH --job-name=postprocess_results
+#SBATCH --time=30-00:00
+#SBATCH --mem=8G
+#SBATCH -c 2
+#SBATCH --nodelist=idmodeling2
```

Comment:
Do these scripts actually need to be run on idm2? Are we pulling from the taxonomy database? Same comment for the scripts below. If not needed, would be better to keep this unspecified so there are more server options

Reply:
this doesn't need to be run on idm2. The R version and the related packages are more important here. I will remove the idm specification here.